e2e/paths/loginAsHmctsAdmin_test.js

```diff
@@ -12,12 +12,53 @@ Before(async (I, caseViewPage, submitApplicationPage) => {
   I.signOut();
 });
 
-Scenario('HMCTS admin can login and add a FamilyMan case number to a submitted case', (I, caseViewPage, loginPage, caseListPage, enterFamilyManPage) => {
+Scenario('HMCTS admin can login and add a FamilyMan case number to a submitted case', (I, caseViewPage, loginPage, enterFamilyManPage, enterOtherProceedingsPage) => {
   loginPage.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
   I.navigateToCaseDetails(caseId);
   I.see(caseId);
-  caseViewPage.goToNewActions(config.addFamilyManCaseNumber);
+  caseViewPage.goToNewActions(config.administrationActions.addFamilyManCaseNumber);
   enterFamilyManPage.enterCaseID();
   I.continueAndSave();
-  I.seeEventSubmissionConfirmation(config.addFamilyManCaseNumber);
+  I.seeEventSubmissionConfirmation(config.administrationActions.addFamilyManCaseNumber);
+
+
+  // Children
```

Comment:
is this still in progress?

Reply:
No, I have just added all the fields that need to be tested and I haven't removed them. 

Removed now, unless someone believe otherwise the e2e tests should be complete.